llvm/lib/Target/PowerPC/PPCInstrInfo.h

```diff
@@ -285,6 +285,9 @@ class PPCInstrInfo : public PPCGenInstrInfo {
   bool isZExt32To64(unsigned Opcode) const {
     return get(Opcode).TSFlags & PPCII::ZExt32To64;
   }
+  bool isMemOp(unsigned Opcode) const {
+    return get(Opcode).TSFlags & PPCII::MemOp;
```

Comment:
Might this be more accurately expressed as "takes register+immediate memory operand"? `isMemOp` seems to simply translate as "accesses memory", which also applies to "register+register" (a.k.a. "indexed") memory operations.

Reply:
I figured register+immediate form is a memory operand because it's a single operand that accesses memory. While the register+register form is a memory operation, it is comprised of two operands. But, I agree it's unclear. I could rename it to isMemriOp (for memory register immediate operand) or something similar. The Tablegen files use "memri" and "memrr" to differentiate.

Reply:
The issue with the "isMemOp" name is that "Op" in this context (combined with "is") means "operation"/"opcode" not "operand". I think "isMemriOp" (as you suggest) or "takesMemriOperand" would be fine.